packages/react-interpolate-components/src/index.es6

```diff
@@ -0,0 +1,128 @@
+/**
```

Comment:
Oh, I had gotten that from [this article](http://mammal.io/articles/using-es6-today/), from which I learned how to trigger transpilation on `npm prepublish`. And he pointed this out about using a `.es6` extension:
> The main reason for the .es6 extension, rather than the perhaps- more-correct .es is that GitHub automatically detects it and applies the correct syntax highlighting

It made a little more sense in the separate repo though, so I could change it here.